### ISSUE TITLE
Vectors not getting parsed on TensorFlow Projector (TripletSemiHardLoss tutorial)

### ISSUE THREAD

**Issue by sayakpaul** [tensorflow/addons#1764]
Hi,

I am referring to this tutorial on TripletSemiHardLoss: https://www.tensorflow.org/addons/tutorials/losses_triplet. After I download the `vecs.tsv` and `meta.tsv` files and head over to https://projector.tensorflow.org/ and try uploading the `vecs.tsv` file I am getting the following:

![image](https://user-images.githubusercontent.com/22957388/80799315-2b07be00-8bc4-11ea-8639-d218011f3dc6.png)

It's been more than 30 minutes, my internet connection is fine. Is it possible to shed some light as to why it might happen? I understand that there might be some erroneous/corrupted entries while preparing the `vecs.tsv`file but I don't think that's the case here.

**Comment by sayakpaul** [comment]
### Update

So, instead of letting Colab download the `.tsv`s for me, I manually downloaded them and it worked fine. Tested it thrice and seems like there's something wrong with automatic download with `files.download`. 

Although I can now upload the embeddings and meta-data to the projector I am unable to reproduce the clustered diagram using UMAP. It would be great if you could provide the hyperparameter values.

**Comment by seanpmorgan** [comment]
Hi @sayakpaul! Apologies for the delayed response... So are you looking for the exact same cluster diagram or something similar (it's not a deterministic training so it won't always be the same)?

I've ran the tutorial and uploaded my embeddings here:
https://drive.google.com/open?id=1ruK9P7LQHVMwvKlZYGi2NAawSxs99Z8g

I ran UMAP with default settings (15 neighbors and 2D) to get a pretty good clustering:

![image](https://user-images.githubusercontent.com/18154355/82854537-b714b900-9ed6-11ea-86ca-58bc9d2e5bea.png)


![image](https://user-images.githubusercontent.com/18154355/82854555-c6940200-9ed6-11ea-9540-084870a5dd9b.png)

Closing but feel free to continue discussion here if there are any further questions or ways you think we can improve the tutorial.
